vue/src/util/loki.ts

```diff
@@ -0,0 +1,113 @@
+export const logParse = {
+  equalLabels: (
+    query: string,
+    preTags: string,
+    postTags: string,
+    keyValue: [string, string],
+    op: string,
+  ) => {
+    if (op === '=') {
+      return `${preTags || ''}{}${postTags || ''}`
+    } else if (op === '!=') {
+      const [key, value] = keyValue
+      return `{${key}!="${value}"}`
+    } else {
+      return query
+    }
+  },
+  splitLabels: (query: string): any[] =>
+    query
+      ?.match(/[^{\}]+(?=})/g, '$1')
+      ?.map((m) => m.split(','))
+      ?.flat() || [], // fix removing from array
+  labelAdd: (op: string, keySubtValue: string, keyValue: string): string => {
+    let labelmod = op === '!=' ? keySubtValue : keyValue
+    return labelmod || ''
+  },
+  valueAdd: (op: string, label: string, value: string, keySubtValue: string): string => {
```

Comment:
Nitpick: any reason you reverse the order of words? E.g. `logParse` instead of `parseLog`, `valueAdd` instead of `addValue` etc...

Reply:
- fixed words order at functions